dotCMS/src/main/java/com/dotcms/api/system/event/Payload.java

```diff
@@ -46,12 +46,16 @@ public Payload(final Object data,
 				   final Visibility visibility,
 				   final String visibilityId) {
 
-		this.type = data.getClass().getName();
+		if(null == data){
+			this.type = Object.class.getName();
```

Comment:
This case shouldn't happen, we should have a constraint to avoid data equals null.
The developer should sends something, at least an void or empty string